### PR TITLE
Task #171108 fix: To add triggers in tjvendors

### PR DESCRIPTION
**Pull request by ChavanAnkitaMahesh**

### CODE REVIEW

File: src/plugins/tjvendors/tjvendors.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	 * Method is called after payout is added.
+	 *
+	 * @param   array  $data  Holds the event data.
+	 * @param   boolean  $isNew  True if payout is stored.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function TjVendorOnAfterPayoutAdd($data, $isNew)
```
Comment (manojLondhe) [inline src/plugins/tjvendors/tjvendors.php:58]: rename as
`TjVendorOnAfterVendorSave`

Comment (manojLondhe) [reply src/plugins/tjvendors/tjvendors.php:58]: use lowercase `method` names

File: src/plugins/tjvendors/tjvendors.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	 * Method is called after vendor data is stored in the database.
+	 *
+	 * @param   array    $data   Holds the new vendor data.
+	 * @param   boolean  $isNew  True if a new vendor is stored.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function TjVendorOnAfterVendorSave($data, $isNew)
```
Comment (manojLondhe) [inline src/plugins/tjvendors/tjvendors.php:42]: use lowercase method names

File: src/plugins/tjvendors/language/en-GB/en-GB.plg_tjvendors_tjvendors.ini [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (manojLondhe) [inline src/plugins/tjvendors/language/en-GB/en-GB.plg_tjvendors_tjvendors.ini:1]: why blank file?
add

```
PLG_TJVENDORS_XML_DESCRIPTION="TJVendors Plugin"
PLG_TJVENDORS="TJVendors plugin"
```

**Review by manojLondhe — CHANGES_REQUESTED** [review]
@snehal0904 @ChavanAnkitaMahesh see comments

File: src/plugins/tjvendors/tjvendors.php (hunk 1 above)
Comment (ChavanAnkitaMahesh) [reply src/plugins/tjvendors/tjvendors.php]: Sir this method name is suggested by snehal mam

**Pull request merged**